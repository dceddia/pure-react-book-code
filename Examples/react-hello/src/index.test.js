jest.mock('react-dom', () => ({render: jest.fn()}))

import React from 'react';
import { HelloWorld } from './index';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer.create(
    <HelloWorld/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
