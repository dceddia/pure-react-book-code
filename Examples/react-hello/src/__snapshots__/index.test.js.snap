// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <span>
    Hello
  </span>
   
  <span>
    World
  </span>
  !
</div>
`;
